### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by SREELAKSHMI-NAYATH**
*Note*: Before submitting this pull request, please review our [contributing
guidelines](https://docs.celeryproject.org/en/master/contributing.html).

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -70,7 +70,7 @@ an older version of Celery:
 
 - Python 2.6: Celery series 3.1 or earlier.
 - Python 2.5: Celery series 3.0 or earlier.
-- Python 2.4 was Celery series 2.2 or earlier.
+- Python 2.4 was the Celery series 2.2 or earlier.
```
Comment (georgepsarakis) [inline README.rst]: I guess it could become consistent with the other two list items' format: `Python 2.4: ...`

**Comment by auvipy** [comment]
your corrections are not correct.

File: README.rst [hunk 2]
```diff
@@ -215,7 +215,7 @@ database connections at ``fork``.
 Documentation
 =============
 
-The `latest documentation`_ is hosted at Read The Docs, containing user guides,
+The `latest documentation`_ is hosted at reading The Docs, containing user guides,
```
Comment (georgepsarakis) [inline README.rst]: The name of the service is indeed `Read the Docs`: 
- https://readthedocs.org/projects/celery/
- https://celery.readthedocs.io/en/latest/

File: README.rst [hunk 3]
```diff
@@ -261,34 +261,34 @@ The following bundles are available:
 Serializers
 ~~~~~~~~~~~
 
-:``celery[auth]``:
+: ``celery[auth]``:
```
Comment (georgepsarakis) [inline README.rst]: Could you please explain the new format? It does not display so well in the [preview](https://github.com/celery/celery/pull/5787/files?short_path=88b99bb#diff-88b99bb28683bd5b7e3a204826ead112), but I am not sure it is rendered 100% accurately.

File: README.rst [hunk 4]
```diff
@@ -261,34 +261,34 @@ The following bundles are available:
...
 ~~~~~~~~~~~
 
-:``celery[eventlet]``:
+: ``celery[eventlet]``:
     for using the ``eventlet`` pool.
 
-:``celery[gevent]``:
-    for using the ``gevent`` pool.
+: ``celery[gevent]``:
+    for using the ``event`` pool.
```
[the remark this answers is not in the document]
Comment (georgepsarakis) [reply README.rst]: Nice catch, no I think it should remain `gevent`.

**Comment by auvipy** [comment]
I closed this as it was apparent the contributor used auto-formatting tool  to change the readme wordings

**Pull request closed without merging**